doc/flame/rendering/images.md

```diff
@@ -10,7 +10,10 @@ flutter:
     - assets/images/enemy.png
 ```
 
-It has to be png files and they can have transparency.
+Images can be in any format supported by Flutter, which include: JPEG, WebP, PNG, GIF, animated GIF,
+animated WebP, BMP, and WBMP. Other formats would require additional libraries. For example, SVG
```

Comment:
is this true of all platforms? (mobile, web, desktop)

Reply:
Here's what the Flutter docs say:
```dart
// Update this list when changing the list of supported codecs.
/// {@template dart.ui.imageFormats}
/// JPEG, PNG, GIF, Animated GIF, WebP, Animated WebP, BMP, and WBMP. Additional
/// formats may be supported by the underlying platform. Flutter will
/// attempt to call platform API to decode unrecognized formats, and if the
/// platform API supports decoding the image Flutter will be able to render it.
/// {@endtemplate}
```

So this looks like a minimal set that's supported on all platforms, plus some additional formats may be supported on each specific platform.